exporter/coralogixexporter/internal/coralogixpb/coralogixpb.go

```diff
@@ -0,0 +1,847 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
There should be instructions on how to regenerate this file on necessity.

@open-telemetry/collector-contrib-approvers, do you think it would make sense to have the originating proto file here as well?

Reply:
I think so long as there is a make target that can generate this code and that target includes the download of the proto, it's fine for it not to be in the repo. The `genproto` target in the collector core downloads the zip file of the proto before running `protoc`.

@ofirshmuel can you add something similar here? https://github.com/open-telemetry/opentelemetry-collector/blob/a1c539a3fe0ef4f94e5cfe5646faf16ab6a62f43/Makefile#L255

Reply:
Any idea where to find the orignal proto files?
I noticed they are not in https://github.com/coralogix/opentelemetry-cx-protobuf-api